source/math/FloatExtension.ooc

```diff
@@ -134,7 +134,7 @@ extend Float {
 		(coefficient, radix) := This decomposeToCoefficientAndRadix(value, 1)
 		power := log10(radix) as Int
 		result := ""
-		if (coefficient != 1.0f)
+		if (Float absolute(coefficient - 1.0f) > 0.000001f)
```

Comment:
ooc supports scientific notation for Float literals, like this:
```ooc
if (Float absolute(coefficient - 1.0f) > 1.0E-6f
```
This makes constants like these easier to read.